samples/app/MeshAssimp.cpp

```diff
@@ -289,9 +577,23 @@ bool MeshAssimp::setFromFile(const Path& file,
                     size_t indicesOffset = outPositions.size();
 
                     for (size_t j = 0; j < numVertices; j++) {
-                        quatf q = mat3f::packTangentFrame({tangents[j], bitangents[j], normals[j]});
+                        float3 normal = normals[j];
```

Comment:
We should at least mention mikktspace here since that's what the spec recommends, and that's what everybody uses.